src/main/java/core/basesyntax/AbstractFigure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class AbstractFigure implements Figure {
```

Comment:
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {
```

Create interfaces AreaCalculator and Drawable with methods for getting area and draw information about figures